tests/listusers/listusers_test.go

```diff
@@ -259,7 +259,8 @@ type document
 			require.NotEmpty(t, fields["trace_id"])
 			if !test.expectedError {
 				require.NotEmpty(t, fields["datastore_query_count"])
-				require.Len(t, fields, 13)
+				require.Empty(t, fields["dispatch_count"])
```

Comment:
Shouldn't this be _not_ empty?
```suggestion
				require.NotEmpty(t, fields["dispatch_count"])
```

Reply:
This was incorrectly considered empty because it was zero. I fixed this test to incur a DB query and dispatch call. These assertions are now a bit stronger.